dev-python/pyqtgraph/pyqtgraph-0.13.7.ebuild

```diff
@@ -50,6 +53,117 @@ python_prepare_all() {
 	if ! use opengl; then
 		rm -r pyqtgraph/opengl || die
 	fi
+
+	# pyqtgraph will automatically use any QT bindings it finds,
+	# patch the supported Qt versions to only include the ones we have enabled
+	local upstream_supported_qt=(
+		PySide
+		PySide2
+		PySide6
+		PyQt4
+		PyQt5
+		PyQt6
+	)
+	local gentoo_qt=()
+	local is_supported_impl use_qt qt
+	# It does comparisons using these variables; set deselected or unsupported to None to avoid more patching
+	for qt in "${upstream_supported_qt[@]}"; do
+		is_supported_impl=0
+		case ${qt} in
+			PySide6) is_supported_impl=1; use_qt=${qt} ;;
+			PyQt5) is_supported_impl=1; use_qt=${qt:2} ;;
+			PyQt6) is_supported_impl=1; use_qt=${qt:2} ;;
+		esac
+		if [[ "${is_supported_impl}" -eq 0 ]]; then
+			gentoo_qt+=( "${qt^^} = None" )
+			continue
+		fi
+		if use ${use_qt,,}; then
+			gentoo_qt+=( "${qt^^} = '${qt}'" )
+		else
+			gentoo_qt+=( "${qt^^} = None" )
+		fi
+	done
+
+	awk -v qt_string="$(printf "%s\n" "${gentoo_qt[@]}")" -i inplace '
```

Comment:
Uhh, all those `awk` s feel to me to be simpler to read if written in Python. On the other hand, I don't want to write them, so as long as it works...

-----

Also, does it make it easier if the `qt_string` is passed through STDIN to awk?